test/ode/ode_extrapolation_tests.jl

```diff
@@ -17,14 +17,14 @@ for i = 1:2
   prob = probArr[i]
   # Order Convergence test
   for j = 1:4
-      sim = test_convergence(dts,prob,RichardsonEuler(j,j,j))
```

Comment:
Was this file ever added to runtests.jl? Might've gotten missed.

Reply:
My mistake. I missed it.
This test file has both regression tests and convergence tests. Which group should it belong?

Reply:
put it into convergence tests III